crates/biome_service/src/workspace/server.rs

```diff
@@ -407,32 +407,15 @@ impl WorkspaceServer {
         let opened_by_scanner = reason.is_opened_by_scanner();
 
         // Second-pass parsing for HTML files with embedded JavaScript and CSS content
-        let (embedded_scripts, embedded_styles) = if let Some(file_source) = self.get_source(index)
+        let (embedded_scripts, embedded_styles) = if let Some(DocumentFileSource::Html(_)) =
```

Comment:
if-let-chains ✨

Reply:
Don’t we need to update our `toolchain.toml`?